api/src/devcomp/domain/models/element/QCM.js

```diff
@@ -22,9 +29,10 @@ class QCM extends Element {
     }
   }
 
-  #assertProposalsIsAnArray(proposals) {
-    if (!Array.isArray(proposals)) {
-      throw new ModuleInstantiationError('The proposals should be in a list');
+  #assertSolutionsAreExistingProposals(solutions, proposals) {
+    const proposalIds = proposals.map((proposal) => proposal.id);
+    if (solutions.every((solution) => proposalIds.includes(solution.id))) {
```

Comment:
**Remarque**
C'est pas l'inverse dans ce cas ? 
```suggestion
    if (!(solutions.every((solution) => proposalIds.includes(solution.id)))) {
```

Reply:
On vient de tester avec @dianeCdrPix,  les solutions sont déjà des ids 😄 
Le bon code :

```suggestion
    if (!(solutions.every((solution) => proposalIds.includes(solution)))) {
```

Reply:
c'est good?